### PR TITLE
Add "Fix an issue" to the hackathon ideas

### LINKED ISSUE #15: Add "fix an issue" to the Hackathon
Speaking to other folks at the NF summit, there's definitely a view that we should be pushing folks to work on getting a development environment set up and going through their first PR. It's a perfect opportunity to get folks past the energy barrier to contributing.

### PR DESCRIPTION
**Pull request by IAlibay**
Fixes #15

### CODE REVIEW

File: hackathon/fix-an-issue/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Fixing an issue in the MDAnalysis core library
+
+With over [400 issues to deal with](https://github.com/MDAnalysis/mdanalysis/issues)
+we are always looking for new and old contributors to help enhance
+and maintain the [MDAnalysis core library](https://github.com/MDAnalysis/mdanalysis).
+
+
+## Team
+
+The MDAnalysis core developers team (@IAlibay, @lilyminium, @fiona-nauthon,
```
Comment (IAlibay) [inline hackathon/fix-an-issue/README.md]: I just added everyone for now, but maybe that's not a good idea

File: hackathon/fix-an-issue/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+## Choose an issue
+
+We have plenty of issues available for you to work on in our
+[Issue Tracker](). For first time contributors, we encourage
+you to look at our list of [easy bugs]() to fix. Have a chat with us,
+to see which one would suit you best! When you found something, please
+let us know in the issue that you are working on it so that other
+attendees do not work on the same thing!
+
+Our recommended pick is the [doctest MEGA-issue]() which involves
```
Comment (IAlibay) [inline hackathon/fix-an-issue/README.md:36]: This would be significantly easier for us to review.

File: hackathon/fix-an-issue/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+development environment, make a fork of the MDAnalysis codebase
+and then contribute the code back using a [Pull Request]().
+
+Our [User Guide]() has a handy [contributors guide]() which you can
+follow for all the information you need to achieve this. Feel free
+to ask any members of the team for help if you encounter any issues
+gettting started.
+
+
+## Choose an issue
```
Comment (IAlibay) [inline hackathon/fix-an-issue/README.md:27]: Do we need a section on the review process?

Comment (IAlibay) [reply hackathon/fix-an-issue/README.md:27]: Added

**Comment by IAlibay** [comment]
@fiona-naughton can I hand this over to you to get merged?

File: hackathon/fix-an-issue/README.md (hunk 1 above)
Comment (orbeckst) [reply hackathon/fix-an-issue/README.md]: Having names here is good. We'll be helping anyway.

File: hackathon/fix-an-issue/README.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
+# Fixing an issue in the MDAnalysis core library
+
+With over [400 issues to deal with](https://github.com/MDAnalysis/mdanalysis/issues)
+we are always looking for new and old contributors to help enhance
+and maintain the [MDAnalysis core library](https://github.com/MDAnalysis/mdanalysis).
+
+
+## Team
+
+The MDAnalysis core developers team (@IAlibay, @lilyminium, @fiona-nauthon,
```
Comment (orbeckst) [inline hackathon/fix-an-issue/README.md]: fix
```suggestion
The MDAnalysis core developers team (@IAlibay, @lilyminium, @fiona-naughton,
```

**Review by orbeckst — APPROVED** [review]
Looking good, thanks for adding.

(will merge the one fix in)

File: hackathon/fix-an-issue/README.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+## Getting started
+
+In order to contribute to MDAnalysis, you will need to set up a
+development environment, make a fork of the MDAnalysis codebase
+and then contribute the code back using a [Pull Request]().
+
+Our [User Guide]() has a handy [contributors guide]() which you can
+follow for all the information you need to achieve this. Feel free
+to ask any members of the team for help if you encounter any issues
+gettting started.
```
Comment (lilyminium) [inline hackathon/fix-an-issue/README.md]:
```suggestion
getting started.
```

File: hackathon/fix-an-issue/README.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+
+## Choose an issue
+
+We have plenty of issues available for you to work on in our
+[Issue Tracker](). For first time contributors, we encourage
+you to look at our list of [easy bugs]() to fix. Have a chat with us,
+to see which one would suit you best! When you found something, please
```
Comment (lilyminium) [inline hackathon/fix-an-issue/README.md]:
```suggestion
to see which one would suit you best! When you find something, please
```

File: hackathon/fix-an-issue/README.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+let us know in the issue that you are working on it so that other
+attendees do not work on the same thing!
+
+Our recommended pick is the [doctest MEGA-issue]() which involves
+fixing our documentation so that code examples are self-contained
+and should run on their own if you tried them out! Here multiple
```
Comment (lilyminium) [inline hackathon/fix-an-issue/README.md]:
```suggestion
and should run on their own if you try them out! Here multiple
```

File: hackathon/fix-an-issue/README.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+## Make a Pull Request!
+
+Once you have made the suitable code change to fix the issue,
+[open a Pull Request]() and we will review your code change
+and work with you towards getting your contribution merged.
+
+This can sometimes be a slow process, especially when the
+code change is large and there are many Pull Requests. If you
+wish to help, you are welcome to review other's Pull Requests
```
Comment (lilyminium) [inline hackathon/fix-an-issue/README.md]:
```suggestion
wish to help, you are welcome to review other people's Pull Requests
```

**Review by lilyminium — APPROVED** [review]
LGTM, thanks. I'll merge the small typos I found if that's ok

**Comment by lilyminium** [comment]
Thanks for opening the issue and PR @IAlibay, I think fixing an issue is a great introduction to the open source community for new contributors!

**Comment by fiona-naughton** [comment]
lgtm, thanks!

**Pull request merged**